kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,20 @@
     });
   }
 
+  var draftbutton = $('.btn-draft');
+  var url = $('.btn-draft' ).data( 'draft-url' );
+  var draftmessage = $('.draft-message').html();
+  draftbutton[0].addEventListener('click', function() {
+    var both_data = $('#both_form').serializeArray();
+    var doc_data = $('#doc_form').serializeArray();
+    var rev_data = $('#rev_form').serializeArray();
+    var total_data = $.extend(both_data, doc_data, rev_data);
+    var posting = $.post( url, total_data );
+    posting.done(function() {
+      var time = new Date();
+      var message = draftmessage + time
+      $('.draft-message').css('display', 'block').empty().append(message)
```

Comment:
Instead of emptying and appending a new message each time, we can simplify this with `.html()`:
`$('.draft-message').html(message);` would replace the entire inner html of the `div` each time. 

For toggling the display of the element, we can use `.show()` instead of `.css()`. We would probably also want this to come *after* we replace the inner content so we're sure the content is updated before we display the element:

`$('.draft-message').html(message).show();`